index.js

```diff
@@ -1,7 +1,89 @@
+console.log(`I'm ready!`);
+
 // Iteration 1: Names and Input
+const hacker1 = 'Marisha';
+console.log(`The driver's name is ${hacker1}.`);
 
+const hacker2 = 'Tiago';
+console.log(`The navigator's name is ${hacker2}.`)
 
 // Iteration 2: Conditionals
+const driver = hacker1.length;
+const navigator = hacker2.length;
 
-
+if (hacker1.length > hacker2.length) {
```

Comment:
Great job on creating variables of the driver's and navigator's length. Here, you could've used the variables you created!

`if (driver > navigator)`